tokio/tests/rt_common.rs

```diff
@@ -343,24 +317,24 @@ rt_test! {
     #[test]
     #[should_panic]
     fn nested_rt() {
-        let mut rt1 = rt();
-        let mut rt2 = rt();
+        let rt1 = rt();
+        let rt2  = rt();
```

Comment:
nit: wrong formatting here
```suggestion
        let rt2 = rt();
```